### PR TITLE
Remove GNTToken

### PR DESCRIPTION
**Pull request by Krigpl**
TestGNT is not really compatible with GNTW so common interface doesn't make sense anymore. Maintaining TestGNT would be just annoying at this point.

### CODE REVIEW

File: golem/ethereum/token.py [hunk 1]
```diff
@@ -40,12 +37,7 @@ def encode_payments(payments: List[Payment]):
     return args
 
 
-class AbstractToken(object, metaclass=abc.ABCMeta):
```
Comment (etam) [inline golem/ethereum/token.py]:
```python
class GNTWToken:
```

File: golem/ethereum/token.py [hunk 2]
```diff
@@ -40,12 +37,7 @@ def encode_payments(payments: List[Payment]):
...
-    """
-    This is a common interface for token transactions. It hides whether we're
-    using GNT or GNTW underneath.
-    """
-
+class GNTWToken(object):
```
Comment (etam) [inline golem/ethereum/token.py]: class GNTWToken:

File: golem/ethereum/token.py [hunk 3]
```diff
@@ -40,12 +37,7 @@ def encode_payments(payments: List[Payment]):
     return args
 
 
-class AbstractToken(object, metaclass=abc.ABCMeta):
-    """
-    This is a common interface for token transactions. It hides whether we're
-    using GNT or GNTW underneath.
-    """
-
+class GNTWToken():
```
Comment (etam) [inline golem/ethereum/token.py]: class GNTWToken:

**Review by Elfoniok — APPROVED** [review]
So after this we assume that the only thing that user can do with GNT is convert it to GNTW?

**Comment by Krigpl** [comment]
> So after this we assume that the only thing that user can do with GNT is convert it to GNTW?

I wouldn't say that's gonna be the case after this PR. it was always the case with GNTW.

**Pull request merged**